plugins/modules/ec2_instance.py

```diff
@@ -1591,11 +1754,19 @@ def build_instance_tags(params):
 def build_run_instance_spec(params, current_count=0):
     spec = dict(
         ClientToken=uuid.uuid4().hex,
-        MaxCount=1,
-        MinCount=1,
     )
     spec.update(**build_top_level_options(params))
 
+    nb_instances = params.get("count") or 1
```

Comment:
```suggestion
    nb_instances = params.get("count", 1)
```

Reply:
This change would result in nb_instances being None when the parameter isn't set, I don't think that's the desired behaviour